codelists/actions.py

```diff
@@ -440,13 +434,11 @@ def publish_version(*, version):
 @transaction.atomic
 def delete_version(*, version):
     """Delete a version.
-
     If this is the only version, the codelist will also be deleted.
-
     Returns a boolean indicating whether the codelist was deleted.
     """
 
-    assert version.is_under_review
+    # assert version.is_under_review
```

Comment:
Should this be commented out?  We don't want to allow published versions to be deleted, do we?

Reply:
Yes this is commented out as I am trying to create an API endpoint to delete published primis codelists

Reply:
But I will try to find a way to pass it in as an option arg